cmd/zdb/zdb.c

```diff
@@ -5903,6 +5906,10 @@ main(int argc, char **argv)
 		case 'X':
 			dump_opt[c]++;
 			break;
+		case 'Y':
+			zfs_reconstruct_indirect_combinations_max = INT_MAX;
```

Comment:
technically I think the rest of the code uses UINT64_MAX here

Reply:
It gets assigned to a `uint64_t` which latter gets set to `UINT64_MAX` but the global is an `int`.  We could change it to an `unsigned long` and set it to `ULONG_MAX`.  Or update it to take a specific value, which is basically what we do now with `-o`.